### DISCUSSION TITLE
How can I have my docker instance change it's vpn ip address ( I use NordVPN if that matters )

### DISCUSSION THREAD

**Discussion by jacksnodgrass** [haugene/docker-transmission-openvpn#2167] [Q&A]
I don't like using the same VPN IP Address on my transmission docker setup. I stop and then start my docker image every few days. I'd like to keep the same docker image running but tell the container to drop the NordVPN connection, select a new NordVPN server and get a new vpn tunnel going so I get a new IP Address. 

Is there a command I can pass to a running docker container to tell it to drop it's openvpn connection and request a new one... it would need to do the: 
2022-01-23 10:01:08 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/usXXXX.nordvpn.com.udp.ovpn 
thing again.

**Comment by edgd1er** [comment]
Hi,

You could change country, then you will get a new ip.
Change setting in docker-compose.yml, restarted the container.

Or, wait a while the pr #2142 to be merged. there will be a new var NORDVPN_SERVER that will bypass nordvpn api and connect to the server.
Bypassing api you will not get the least used server and could even try to connect to dead server.
